tests/integration/solidity_contracts/PlainOpcodes/test_safe.py

```diff
@@ -1,38 +1,32 @@
 import pytest
+from tests.utils.constants import ACCOUNT_BALANCE
 
 
 @pytest.mark.asyncio
 @pytest.mark.Counter
 @pytest.mark.usefixtures("starknet_snapshot")
 class TestSafe:
     class TestReceive:
-        async def test_should_receive_eth(self, safe, owner, eth):
-            await eth.mint(owner.starknet_address, (int(1e9), 0)).execute(
-                caller_address=owner.starknet_address
-            )
-            await safe.deposit(value=int(1e9), caller_address=owner.starknet_address)
-            assert await safe.balance() == 1e9
+        async def test_should_receive_eth(self, safe, owner):
+            await safe.deposit(value= ACCOUNT_BALANCE, caller_address=owner.starknet_address)
```

Comment:
you definitely **need** to run `make format` (black here) on save

Reply:
gotcha!

Might be we can also add git hooks just like how we have at the RPC? Lemme know, I can create an issue for it.